backend/authentication/forms.py

```diff
@@ -66,12 +76,13 @@ def get_token(self, user):
         return MailAuthBackend.get_token(user=user)
 
     def save(self, commit=True):
+        """redefine just to transmit the login_url in test mode"""
         super().save(commit)
         email = self.cleaned_data["email"]
-        for user in EmailLoginForm.get_users(self, email):
-            context = EmailLoginForm.get_mail_context(self, self.request, user)
-            EmailLoginForm.send_mail(self, email, context)
-            if "test" in sys.argv:
+        for user in MailauthEmailLoginForm.get_users(self, email):
+            context = MailauthEmailLoginForm.get_mail_context(self, self.request, user)
+            MailauthEmailLoginForm.send_mail(self, email, context)
+            if "test" in sys.argv or "backend:test" in sys.argv:
```

Comment:
If `test` is in `sys.argv`, it is as well in `backend:test`

Reply:
`sys.argv` is not a string but the list of command line arguments : `"test"` is in `["python", "manage.py", "test"]` but not in `["npm", "run", "backend:test"]`